### DISCUSSION TITLE
kubernetes.default.svc all namespaces

### DISCUSSION THREAD

**Discussion by elmasoki** [argoproj/argo-cd#9383] [Q&A]
Hi,

We installed ArgoCD  on Openshift using ArgoCD operator.  When we change  namespace  to " All Namespace" for default  kubernetes cluster It turns to argocd namespace automatically.  we had to stop [argocd-operator-controller-manager] pod to keep namespace "All namespace". 

How could we keep it as "all namespace"  without stopping [argocd-operator-controller-manager] . 

thank you in advance.

![image](https://user-images.githubusercontent.com/10141608/168089429-19cb0d7b-b7c5-402f-a3fa-54b417f5a479.png)

**Comment by rishabh625** [comment]
Hi @elmasoki  you are on wrong forum probably you should ask [here](https://github.com/argoproj-labs/argocd-operator/discussions)

**Reply by elmasoki** [reply]
thanks. How can i delete this discussion
